### ISSUE TITLE
TypeError: run() got an unexpected keyword argument 'clear_cache'

### ISSUE THREAD

**Issue by ARamsey118** [papis/papis#110]
Output of ```papis -v --debug```

```
25-DEBUG:papis:DEBUG MODE FOR DEVELOPERS ON
25-DEBUG:papis:Imported commands
Traceback (most recent call last):
  File "/home/andrew/.local/bin/papis", line 11, in <module>
    load_entry_point('papis==0.7.3', 'console_scripts', 'papis')()
  File "/home/andrew/.local/lib/python3.7/site-packages/papis/main.py", line 22, in main
    papis.commands.default.run()
  File "/home/andrew/.local/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/andrew/.local/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/andrew/.local/lib/python3.7/site-packages/click/core.py", line 1043, in invoke
    return Command.invoke(self, ctx)
  File "/home/andrew/.local/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/andrew/.local/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
TypeError: run() got an unexpected keyword argument 'clear_cache'
```

This error occurs when simply running papis with no arguments using python3.5 on Ubuntu 16.04 from master, as well as OS X with python3.7. It does not happen on Arch running python3.7. On OS X and Ubuntu, it can be fixed by swapping the order of `"--clear-cache"` and `"--cc"`. However, this causes the error to appear on Arch.

**Comment by alejandrogallo** [comment]
[GitMate.io](https://gitmate.io) thinks the contributors most likely able to help are @alejandrogallo, and @bluehood.

**Comment by alejandrogallo** [comment]
Hi! Thanks for the report.
This seems really odd, can you check the version of click ? 
You just have to input this in a python prompt
```python
import click
print(click.__version__)
```
Thanks!

**Comment by ARamsey118** [comment]
Click is version 7.0 on the computers with the error and 6.7 on the one without. Upgrading 6.7 to 7 causes it to stop working, and a downgrade to 6.7 fixes it once again.

**Comment by ARamsey118** [comment]
I believe the error is a result of [this](https://github.com/pallets/click/pull/794), which will cause the first long option to be selected, but `cc` is the second long option.

**Comment by zamlz** [comment]
Hi, I can also confirm this bug. Just tried setting up the program today. My click version is also `7.0`.

**Comment by alejandrogallo** [comment]
I just fixed, now it's working for click 7.0 and click 6.7

Thank you two for checking on this! This makes papis possible.

**Comment by ARamsey118** [comment]
Thanks for fixing it! It's working well for me
